ckanext/data_qld_theme/templates/base.html

```diff
@@ -2,8 +2,10 @@
 
   {% block meta %}
     {{ super() }}
-    {% if c.pkg_dict and c.pkg_dict['organization'] and c.pkg_dict['organization']['title'] %}
-        <meta name="DCTERMS.creator" content="c=AU; o=The State of Queensland; ou={{ c.pkg_dict['organization']['title'] }}" scheme="AGLSTERMS.GOLD" />
+    {% if c.controller in ['dataset', 'package', 'ckanext.data_qld.controller:DataQldDataset'] %}
```

Comment:
@ThrawnCA  did you do a tweak here to make this work recently?

Reply:
Yes, I did. We shouldn't rely on specific controllers or actions, just the presence of the necessary org info.

Reply:
Happy to adjust as suggested, however as an FYI...

The reason for implementing it this way is based on the original requirements, whereby this meta tag should only appear on dataset detail and resource detail pages for the purpose of GTM tracking.

It may not be an issue if this meta tag appears on organisation profile pages, I don't know - but it would be worth checking with whoever manages the GTM account before making a change where it appears on any page that contains organisation information.